src/screens/CreatePost/index.js

```diff
@@ -797,7 +797,7 @@ const CreatePost = () => {
           )}
           <Gap style={styles.height(25)} />
           <Button styles={styles.btnPost(typeUser)} disabled={isButtonDisabled()} onPress={postV2}>
-            Post
+            {params.isCreateCommunity ? 'Post & Create Community' : 'Post'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text for the button is now conditionally rendered based on the `params.isCreateCommunity`. This is a good way to reuse the component for different purposes. However, similar to the previous comment, make sure `params.isCreateCommunity` is always defined and is a boolean.
```diff
- {params.isCreateCommunity ? 'Post & Create Community' : 'Post'}
+ {params?.isCreateCommunity === true ? 'Post & Create Community' : 'Post'}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->